run-pylint.py

```diff
@@ -104,14 +111,16 @@ def cleanup_results_dict(r, sym):
     "--load-plugins", "pylint.extensions.eq_without_hash",
 ]
 
-def pylint_project(module_path: str, errorlog: TextIOWrapper, branch_url: str):
+def pylint_project(check_dirs: List[str], errorlog: TextIOWrapper, branch_url: str):
 
     pylint_overview = []
     pylint_results = []
-    glob_pattern = os.path.join(module_path, "**", "*.py")
+    pylint_paths = []
+    check_patterns = [p + "/**/*.py" for p in check_dirs]
+    list(map(lambda x: pylint_paths.extend(glob(x, recursive=True)), check_patterns))
```

Comment:
Personally I prefer to use a loop to perform an action instead of a mapping and then discarding the result, I think this makes the code easier to follow